prow/plugins/trigger/trigger.go

```diff
@@ -36,6 +36,11 @@ import (
 const (
 	// PluginName is the name of the trigger plugin
 	PluginName = "trigger"
+
+	nonTrustedNotMember                                  = "User is not a member of the org."
+	nonTrustedNotMemberNotCollaborator                   = "User is not a member of the org and is not a collaborator. Either will make the user trusted."
+	nonTrustedNotMemberNotSecondaryMember                = "User is not a member of the org or the trusted (secondary) org"
+	nonTrustedNotMemberNotSecondaryMemberNotCollaborator = "User is not a member of the org or the trusted (secondary) org and is not a collaborator. Any of these being true will make the user trusted."
```

Comment:
There seems to be redundancies in these consts as well. It looks like there's 3 main reasons why users aren't trusted, might be worthwhile to turn those into enums and add a func that constructs a reason based on 0 or more of those reasons.

something like: https://play.golang.org/p/S27-bXXAhW-

Reply:
Added. That's really cool.